praw/reddit.py

```diff
@@ -542,6 +546,31 @@ def generator(url):
 
         return generator(url)
 
+    def _objectify_request(
+        self,
+        method: str,
```

Comment:
Since we're making a new private method we technically don't need to document it. However, since we have, let's please sort the parameters and the corresponding documentation for them.